tests/unit/test_openvswitch.py

```diff
@@ -214,6 +225,26 @@ def test_summary_ovn_tunnels_no_remotes(self):
             actual = self.part_output_to_actual(inst.output)['tunnels']
             self.assertEqual(actual, expected)
 
+    @utils.create_data_root({'sos_commands/ovn_central/microovn.ovn-nbctl_'
+                             '--no-leader-only_list_Connection':
+                             OVNDB_TABLE_CONNECTION,
+                             'sos_commands/ovn_central/microovn.ovn-sbctl_'
+                             '--no-leader-only_list_Connection':
+                             OVNDB_TABLE_CONNECTION})
+    @mock.patch.object(OVNBase, 'is_ovn_central', True)
+    def test_summary_ovn_central_db(self):
+        expected = {'ovn': {
+                        'nbdb': {'routers': 0, 'switches': 0,
```

Comment:
There are no routers or switches listed in the Connection command on my system, is this correct? Same applies to SB

Reply:
yeah im not entirely happy with this output as it is a mix of db settings and values that are being generated by hotsos. i think ill separate them in the output to make it more clear.

Reply:
hopefully looks better now:

```
openvswitch:
  snaps:
    - microovn 24.03.2+snapa2c59c105b (-)
  config:
    nbdb:
      inactivity_probe: '[]'
      max_backoff: '[]'
    sbdb:
      inactivity_probe: '[]'
      max_backoff: '[]'
  ovn:
    nbdb:
      routers: 2
      switches: 4
    sbdb:
      chassis: 5
      ports: 11
      router-gateways: 0
```